codalab/lib/bundle_cli.py

```diff
@@ -3853,7 +3852,7 @@ def do_uls_command(self, args):
         Search for specific users.
         """
         client = self.manager.current_client()
-        users = client.fetch('users', params={'keywords': args.keywords})
+        users = client.fetch('users', params={'keywords': args.keywords or ''})
```

Comment:
Can you add a comment that this code makes `cl uls` with no arguments send an empty query?